### PR TITLE
Feature/apply design profile card edit screen roughly

### LINKED ISSUE #112: Apply profile card edit screen design roughly
| current | ideal |
|--------|--------|
|  | image | 

https://www.figma.com/design/XUk8WMbKCeIdWD5cz9P9JC/DroidKaigi-2024-App-UI?node-id=54795-26746&t=jBceLP15jxDiPfMP-0

### PR DESCRIPTION
**Pull request by hiroaki404**
## Issue
- close #112

## Overview (Required)
- I applyed design profile card edit screen roughly.

I applyed only design. I think there is some remaining task.  I will not implement below in this PR.
- Implements presenter or data layer
- Text field validation
- I think there is much room for refactoring.

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)
Before | After
:--: | :--:
<video src="https://github.com/user-attachments/assets/f77efb64-a7dc-486d-a931-ef70206b8e6d" width="300" > | <video src="https://github.com/user-attachments/assets/b0127891-70c3-4c5d-9aaa-40ff30f59739" width="300" >

### CODE REVIEW

**Comment by hiroaki404** [comment]
Sorry for the delay in working on this. Please review!

File: feature/profilecard/src/commonMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardScreen.kt [hunk 1]
```diff
@@ -214,80 +247,239 @@ internal fun EditScreen(
...
+
+@Composable
+private fun ThemeImage(
+    isSelected: Boolean,
+    theme: ProfileCardTheme,
+    modifier: Modifier = Modifier,
+    onClickImage: (ProfileCardTheme) -> Unit,
+) {
+    val colorMap = buildMap {
+        put(ProfileCardTheme.Iguana, Color(0xFFB4FF79))
```
Comment (takahirom) [inline feature/profilecard/src/commonMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardScreen.kt:427]: Can we use ProfileCardScreenTheme here?

Comment (takahirom) [reply feature/profilecard/src/commonMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardScreen.kt:427]: Sorry we can improve after merging 🙏

Comment (hiroaki404) [reply feature/profilecard/src/commonMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardScreen.kt:427]: Sorry, I didn't realize it existed 😅

**Pull request merged**